maths/prime_numbers.py

```diff
@@ -97,9 +97,9 @@ def benchmark():
     from timeit import timeit
 
     setup = "from __main__ import slow_primes, primes, fast_primes"
-    print(timeit("slow_primes(1_000_000_000_000)", setup=setup, number=1_000_000))
-    print(timeit("primes(1_000_000_000_000)", setup=setup, number=1_000_000))
-    print(timeit("fast_primes(1_000_000_000_000)", setup=setup, number=1_000_000))
+    print(timeit("slow_primes(1_000)", setup=setup, number=1_000))
+    print(timeit("primes(1_000)", setup=setup, number=1_000))
+    print(timeit("fast_primes(1_000)", setup=setup, number=1_000))
```

Comment:
Please revert these three lines.  pytest does not run the code in `__main__()` so it does not run the benchmarks.